src/context.py

```diff
@@ -0,0 +1,69 @@
+Features = {"CASEMAPPING": "rfc1459", "CHARSET": "utf-8", "STATUSMSG": {"@", "+"}, "CHANTYPES": {"#"}} # IRC server features (these are defaults)
```

Comment:
We only handle very specific values of this, so we should have an Enum encoding the values we support. If the ircd sends an unsupported value, we'll just default it.